src/lib/common/versionInfo.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_LIB_COMMON_VERSIONINFO_H_

```

Comment:
Please add a line about the change involved in this PR in the CHANGES_NEXT_RELEASE file. That file is used during the releasing process in order to generate new Orion version (e.g. 1.9.0).

Suggestion:

> - Add: add release_date and doc fields to GET /version output to align with FIWARE scheme (#2971)

Reply:
Fixed in e88551b

(This in a example on where to put the "Fixed in" comments, i.e. in comment threads initiated by a *code line comment* instead of *a general comments*)